README.md

```diff
@@ -273,41 +273,43 @@ datalad push --to $github_sibling_name
 
 ## Add an existing R2 sibling to your local repository
 
-If someone already set up the R2 sibling, we still need 
-to add it to our local repository. When we pull from GitHub our local repository
-is already aware of the sibling, but we might not be able to see it yet.
+If someone already set up the R2 remote, we still need 
+to enable the sibling in our local repository. Our local repository
+should be already aware of the sibling, but we might not be able to see it yet.
 
 First we check if the R2 sibling has already been added with:
 
 ```shell
 git annex whereis path/to/some/file
 ```
 
-We should now see a list of remotes where the file is stored. One of them will have the name `public-r2`
-and show the url of the R2 bucket.
+Choose a file that ist stored in the datalad remote (e.g. `.nc` or `.csv` files). We should now see a list of remotes 
+where the file is stored. One of them will have the name `public-r2`
+and show the url of the R2 bucket. Note that it could also have another name, if the remote wasn't set up
+as described above.
```

Comment:
```suggestion
Choose a file that is stored in the datalad remote (e.g. `.nc` or `.csv` files). We should now see a list of remotes 
where the file is stored. One of them will have the name `public-r2`
and show the url of the R2 bucket. Note that it could also have another name, if the remote wasn't set up
as described above.
```